mediator/src/main/kotlin/no/nav/dagpenger/soknad/db/SøknadPostgresRepository.kt

```diff
@@ -271,7 +271,7 @@ private fun Session.hentDokumenter(innsendingId: UUID): InnsendingDTO.Dokumenter
                  hoveddokument
             WHERE dokument_v1.innsending_uuid = :innsendingId 
             """.trimIndent(),
-            "innsendingId" to innsendingId
+            mapOf("innsendingId" to innsendingId)
```

Comment:
Var det buggen?

Reply:
Det var en bug jeg introduserte ved et uhell :D Som gjorde at jeg trodde jeg klarte å reprodusere ting, men det var feil bug. Umulig å reprodusere oppførselen i testene, men feiler hver gang i miljø.

Reply:
Vi har også blitt litt lurt i testene fordi deepequalsen ikke deepequalet dokumentene, men hacka inn en visitor for å asserte nå, og i testen så er dokumentene så vitterlig der.